ninja-core/src/main/java/ninja/session/SessionImpl.java

```diff
@@ -139,13 +135,75 @@ public void init(Context context) {
         }
     }
 
+    protected boolean shouldExpire() {
+        if (sessionExpireTimeInMs != null) {
+
+            // Make sure session contains valid timestamp
+            if (!data.containsKey(TIMESTAMP_KEY)) {
+
```

Comment:
Tiny - but please compact the code by using empty lines only sparsely (I know this may not have been the case before, but it's just more concise :) ). Thanks!